notebooks/02_model/als_deep_dive.ipynb

```diff
@@ -87,16 +87,16 @@
   },
```

Comment:
<p>can we make these notebook parameters? then in the test we can just choose a single option for both, will need to make sure the following plotting doesn't fail though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/864/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
Good point. Yea the plotting works with 50% of the movielens 100k dataset